lib/msal-common/src/server/ServerParamsGenerator.ts

```diff
@@ -0,0 +1,257 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AADServerParamKeys, SSOTypes } from "../utils/Constants";
+import { Constants, HeaderNames } from "../utils/Constants";
+import { ScopeSet } from "../request/ScopeSet";
+import { Authority } from "../authority/Authority";
+import { ClientConfigurationError } from "../error/ClientConfigurationError";
+
+export class ServerParamsGenerator {
```

Comment:
Not sure I understand the point of having a class with static members here. This is difficult to maintain and would rather have a single function that does all of this as we had before.

Reply:
This is like utils, and need all functions separate as device code and auth code may have some common, some different params. We want to be able to call them independently. Do not want them in a single function for flexibility.